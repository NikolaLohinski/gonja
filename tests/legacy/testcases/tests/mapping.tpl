{}: {{ {} is mapping }}
simple.intmap: {{ simple.intmap is mapping }}
[]: {{ [] is mapping }}
42: {{ 42 is mapping }}
