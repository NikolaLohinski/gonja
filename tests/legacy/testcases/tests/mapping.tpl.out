{}: True
simple.intmap: True
[]: False
42: False
